pkg/controller/scyllacluster/resource.go

```diff
@@ -543,37 +543,53 @@ func StatefulSetForRack(r scyllav1.RackSpec, c *scyllav1.ScyllaCluster, existing
 							// TODO: unprivileged entrypoint
 							Command: func() []string {
 								cmd := []string{
-									path.Join(naming.SharedDirName, "scylla-operator"),
-									"sidecar",
-									fmt.Sprintf("--feature-gates=%s", func() string {
+									"/usr/bin/bash",
+									"-euEo",
+									"pipefail",
+									"-O",
+									"inherit_errexit",
+									"-c",
+									strings.TrimSpace(`
+echo "Waiting for /mnt/shared/ignition.done" > /dev/stderr
+until [[ -f "/mnt/shared/ignition.done" ]]; do
+  sleep 1;
+done
+
+set -x
+
+# TODO: This is where we should start ScyllaDB directly after the sidecar split #1942 
+exec /mnt/shared/scylla-operator sidecar \
+--feature-gates=` + func() string {
 										features := utilfeature.DefaultMutableFeatureGate.GetAll()
 										res := make([]string, 0, len(features))
 										for name := range features {
 											res = append(res, fmt.Sprintf("%s=%t", name, utilfeature.DefaultMutableFeatureGate.Enabled(name)))
 										}
 										sort.Strings(res)
 										return strings.Join(res, ",")
-									}()),
-									fmt.Sprintf("--nodes-broadcast-address-type=%s", func() scyllav1.BroadcastAddressType {
```

Comment:
why move away from Sprintf?

Reply:
its no longer an array of strings but one string so it's consistent with the fixed flags

Reply:
(ideally all of this would be a go template but that's a change to be discussed separately for sure)